src/views/Login.vue

```diff
@@ -47,8 +47,11 @@ export default {
   }),
   methods: {
     login() {
+      const baseUrl =
+        (window.openpaas && window.openpaas.OPENPAAS_API_URL) || "";
```

Comment:
```suggestion
      const baseUrl = (window.openpaas && window.openpaas.OPENPAAS_API_URL) || "";
```

Reply:
![Screenshot from 2020-08-11 09-17-39](https://user-images.githubusercontent.com/4060207/89849868-a7427200-dbb3-11ea-98de-5649429cd86f.png)

Doing that and the ESLint Prettier of this project will scream :D. So I respected that current ESLint config of this project

Reply:
The `printWidth` rule is probably too small. Well it doesn't really matter anyway.